test/gtest/common/gtest-all.cc

```diff
@@ -4713,7 +4713,6 @@ void PrettyUnitTestResultPrinter::OnTestPartResult(
   switch (result.type()) {
     // If the test part succeeded, or was skipped,
     // we don't need to do anything.
-    case TestPartResult::kSkip:
```

Comment:
This allows to print the "skip message"
In the present case in prints
```
../../../test/gtest/tl/mlx5/test_tl_mlx5.cc:43: SkippedSkipped
no ib devices
```
which gives a clear explanation

this is inspired by
https://github.com/google/googletest/pull/2517

We should consider making gtest a proper third party library, wdyt?